README.md

```diff
@@ -1 +1,5 @@
-# Hello_Microverse_Project
\ No newline at end of file
+# Hello_Microverse_Project
+This is a project to set up a "Hello Microverse" project
+
+# Built With
+HTML CSS
```

Comment:
- [ ] As previously requested, use the template provided by microverse for your readme file.